pom.xml

```diff
@@ -46,8 +46,8 @@
 		</dependency>
 
 		<dependency>
-			<groupId>io.prestosql</groupId>
-			<artifactId>presto-base-jdbc</artifactId>
+				<groupId>javax.validation</groupId>
+				<artifactId>validation-api</artifactId>
```

Comment:
would you please reformat it by removing whitespaces/tabs?

Reply:
done